doc/man3/EVP_DigestInit.pod

```diff
@@ -318,6 +321,27 @@ Similar to EVP_DigestFinalXOF() but allows multiple calls to be made to
 squeeze variable length output data.
 EVP_DigestFinalXOF() should not be called after this.
 
+=item EVP_DigestSerialize() and EVP_DigestDeserialize()
+
+EVP_DigestSerialize() serializes the state of the digest context I<ctx>
+and stores it in I<out>, unless I<out> is NULL.
+Otherwise the maximum necessary size of the output buffer is written to
+the I<outlen> parameter. If I<sig> is not NULL then before the call the
+I<outlen> parameter must contain the length of the I<out> buffer. If the
+call is successful the data is written to I<out> and the amount of data
+written to I<outlen>.
+
+EVP_DigestDeserialize() deserializes the data from I<in> of size I<inlen>
+into the digest context I<ctx>.
+
+These functions does not guarantee importability of state exported by a
+different OpenSSL version and does not guarantee interoperability between
+different providers.
+
+If a provider does not support serialization this function returns 0 and
+an an error code is set on the error stack. The L<ERR_get_error(3)> function
```

Comment:
Is the error code bit necessary?

Reply:
I don't think so

Reply:
The docs would be pretty bloated if we start doing this everywhere.